Show start button in the calendar view

juanlaube opened the pull request:
## What's this?
This adds the start button to the calendar view, and the stop button but without a final design.

### Relationships

Refs #7678 

## Why do we want this?
We want to start time entries from the calendar view, so it can be set as the default view of the app.

## How is it done?
Adding the start and stop buttons.

## :squid: Permissions
🦑

sofokli1 commented:
also every time I change tab it looks like the button pops into view, is that on purpose?

juanlaube commented:
@sofokli1 I fixed the bug with the duplicated button.
Also about the animation when switching tabs, it's something I added but I don't know we will keep it 🤷‍♂️
